packages/reactivity/src/warning.ts

```diff
@@ -0,0 +1,3 @@
+export function warn(msg: string, ...args: any[]) {
+  console.warn(`[Vue warn] ${msg}`, ...args)
```

Comment:
Since `warn` for Vue is present in `runtime-core` binding with the component model, is it ok to introduce this simple version of it in `reactivity`?